ambassador/ambassador/envoy/v2/v2listener.py

```diff
@@ -30,12 +30,43 @@
 from ...ir.ircluster import IRCluster
 from ...ir.irtcpmappinggroup import IRTCPMappingGroup
 
+from ambassador.utils import ParsedService as Service
+
+import logging
```

Comment:
All this logging/argv stuff needs to not be here -- `v2listener.py` is primarily `import`ed, not run from the command line.